portfolio/src/main/webapp/index.html

```diff
@@ -138,10 +182,141 @@ <h5> Feb 2017 - Nov 2018</h5>
             </div>
           </div>
         </div>
-
       </div>
     </section><!-- End Resume Section -->
     </main> <!--End of main-->
+    
+    <!--========= Blog Section ========-->    
+    <section id="blog" class="blog-mf sect-pt4 route">
+      <div class="container">
+        <div class="row">
+          <div class="col-sm-12">
+            <div class="title-box text-center">
+              <h3 class="title-a">
+                Welocme to my Blog
+              </h3>
+              <p class="subtitle-a">
+                I am passionate about Computer Science education and I have recently started blogging on my journey learning new programming languages. Have fun reading!
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-4">
+            <div class="card card-blog">
+              <div class="card-img">
+                <a href="https://minoltandlovu.medium.com/my-journey-learning-javascript-week-2-javascript-fundamentals-part-2-d0ba787f76af"><img src="..." alt="" class="img-fluid"></a>
```

Comment:
If you can, try to break up these long single lines into shorter (in horizontal length) multiple lines.

https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping